app/src/main/java/ai/elimu/appstore/model/Application.java

```diff
@@ -14,11 +18,15 @@
 import ai.elimu.model.enums.content.LiteracySkill;
 import ai.elimu.model.enums.content.NumeracySkill;
 
-import java.util.Set;
-
 @Entity
 public class Application {
 
+    @Transient
```

Comment:
Is there any way we can solve this issue without introducing transient variables to the `Application` entity. Try to not mix additional logic/functionality into the entities, since they should be as clean as possible and be similar to https://github.com/elimu-ai/model/blob/master/src/main/java/ai/elimu/model/gson/admin/ApplicationGson.java